components/user/UserTabs.js

```diff
@@ -1,153 +1,157 @@
-import { BiSortAlt2 } from "react-icons/bi";
-import Link from "../Link";
+import { BiSortAlt2 } from 'react-icons/bi';
+import Link from '../Link';
```

Comment:
Please allow our coding standards, these changes are not required and make the project inconsistent